src/orders/components/OrderCustomerAddressesEditDialog/OrderCustomerAddressEdit.tsx

```diff
@@ -48,7 +49,8 @@ const OrderCustomerAddressEdit: React.FC<OrderCustomerAddressEditProps> = props
     formErrors,
     onChangeFormAddress,
     onChangeFormAddressCountry,
-    onEdit
+    onEdit,
+    hideCard = false
```

Comment:
Not a big fan of negative based props like this, I think it's always less complicated to think in positives, especially when adding negations later. Right now I can see it's used like so:
```hideCard: !hasCustomerChanged``` and ```!hideCard``` -> using it with negations every everywhere
We could change it to `showCard` and then do
```showCard: hasCustomerChanged``` and ```showCard```